source/common/json/config_schemas.cc

```diff
@@ -220,6 +220,11 @@ const std::string Json::Schema::CLIENT_SSL_NETWORK_FILTER_SCHEMA(R"EOF(
         "type" : "integer",
         "minimum" : 0,
         "exclusiveMinimum" : true
+      },
+      "request_timeout_ms" : {
```

Comment:
Revert all changes to the v1 schemas. Basically, v1 should be hard coded to 1s.

Reply:
Addressed in 0ecfc41.